### PR TITLE
removing jes tests closes #81

### LINKED ISSUE #81: Fix Centaur Optional Test cases
So that we can start moving JES-specific tests to Centaur-private

### PR DESCRIPTION
**Pull request by ruchim**
removing jes tests

### CODE REVIEW

**Comment by ruchim** [comment]
@geoffjentry @cjllanwarne These tests are the only candidates that jumped out at me for removal, but I'm open to whatever else shouldn't be publicly available

File: src/main/resources/application.conf [hunk 1]
```diff
@@ -12,4 +12,5 @@ centaur {
   # case will be in a subdirectory named FOO with files FOO.wdl, FOO.inputs, and FOO.options. This is not
   # particularly robust at the moment.
   standardTestCasePath: "src/main/resources/standardTestCases"
+  optionalTestPath: "/Users/rmunshi/centaur-private/jesTestCases"
```
Comment (cjllanwarne) [inline src/main/resources/application.conf]: I'm pretty sure this doesn't belong in the repo 😄

Comment (cjllanwarne) [inline src/main/resources/application.conf]: I suggest commenting out the option name and blanking the value, e.g.

```
# optionalTestPath: "/path/to/test/directory"
```

**Comment by cjllanwarne** [comment]
Otherwise, 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/82/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/application.conf (hunk 1 above)
Comment (geoffjentry) [inline src/main/resources/application.conf]: To a naive user that might conflict with the text in the README

Comment (ruchim) [inline src/main/resources/application.conf]: You're right, that wasn't supposed to be there! It shall disappear.

File: src/main/resources/application.conf [hunk 2]
```diff
@@ -12,4 +12,5 @@ centaur {
   # case will be in a subdirectory named FOO with files FOO.wdl, FOO.inputs, and FOO.options. This is not
   # particularly robust at the moment.
   standardTestCasePath: "src/main/resources/standardTestCases"
+  # optionalTestPath: "/path/to/test/directory"
```
Comment (cjllanwarne) [inline src/main/resources/application.conf]: Since @geoffjentry wants to match the README: `# optionalTestPath=/some/path/to/tests`

**Comment by geoffjentry** [comment]
@cjllanwarne Actually what I meant re the README is that the README doesn't explain that it could also just be set via conf file, instead it specifies the command line arg

**Comment by cjllanwarne** [comment]
@geoffjentry see #83

**Comment by ruchim** [comment]
@geoffjentry just waiting on your thumb :)

File: run_tests_parallel.sh [hunk 3]
```diff
@@ -10,6 +10,6 @@ THREADS=${1:-3}
 
 echo "Running tests with ${THREADS}-way parallelism"
 
-sbt test:compile
+sbt -Dcentaur.optionalTestPath=/Users/rmunshi/centaur-private/jesTestCases test:compile
```
Comment (geoffjentry) [inline run_tests_parallel.sh]: This shouldn't be in centaur-non-private

**Comment by geoffjentry** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/82/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

**Pull request merged**